### ISSUE TITLE
Vectortile always empty

### ISSUE THREAD

**Issue by allspatial** [mapnik/node-mapnik#693]
Hi, 

I am trying to create a vectortile using the code as follows:

```javascript
var geojson = {"type":"FeatureCollection","features":[{"type":"Feature","geometry":{"type":"LineString","coordinates":[[6199977.26593086,-481416.046458018],[6199977.45879732,-481419.265669088],[6199977.06254467,-481423.28313843],[6199976.53751613,-481427.406803341],[6199974.99395134,-481437.490110601],[6199974.50134937,-481440.919391656],[6199967.76586181,-481506.047681743]]},"properties":{}}]};
var vt = new mapnik.VectorTile(10, 669, 525);
vt.addGeoJSON(JSON.stringify(geojson), 'road', {});
console.log(vt.names());
console.log(vt.toGeoJSONSync('road'));
```

The output for the layername(s) in the second last line is always an empty array. The last line would fail with an error saying "Layername 'road' not found". I am using node4-LTS with mapnik 3.5.13 (and mapnik-vector-tile 1.2.0) running on OS X El Capitan (but having the same issue on Ubuntu 14.04.).

Thanks for your help!

**Comment by mapsam** [comment]
Hey there @allspatial! Before going through any Node Mapnik API issues, I'm noticing your GeoJSON coordinates don't reflect that of WGS84 coordinate reference system. It seems they are in meters, probably a variation of web-mercator.

I'd try updating that first, then see if you're able add data to a vector tile.

**Comment by mapsam** [comment]
If you're using GDAL, here's the reprojection script I tried, assuming the original projection was Web Mercator 

```
ogr2ogr -f "GeoJSON" wgs84.geojson -t_srs EPSG:4326 -s_srs EPSG:3857 original.geojson
```

I tried with the new GeoJSON and got similar results as yourself, with no `road` layer being found. That made me curious if the zxy tile coordinate was correct. I grabbed one of the coordinates from the goejson and used node-spherical mercator to calculate the z10 tile x and y, which was different than the tile coordinate you have. This means the vector tile will contain no data since the geojson doesn't overlap the tile. The correct tile coordinates are `10, 670, 524`. 

Lastly, don't forget to register the default plugins when using mapnik! https://github.com/mapnik/mapnik/wiki/PluginArchitecture

Here's my working script script:

```
var mapnik = require('mapnik');
var geojson = {} ... // newly reprojected geojson
var vt = new mapnik.VectorTile(10, 670, 524);
vt.addGeoJSON(JSON.stringify(geojson), 'road');
console.log(vt.names()); // => ['road']
console.log(vt.toGeoJSONSync('road')); // => geojson converted from tile coordinates
```
